gloss/Graphics/Gloss/Internals/Interface/Backend/GLFW.hs

```diff
@@ -152,14 +167,15 @@ openWindowGLFW ref FullScreen
         let sizeX = GLFW.videoModeWidth (fromJust vmode)
         let sizeY = GLFW.videoModeHeight (fromJust vmode)
 
-        win <- GLFW.createWindow 
+        win <- GLFW.createWindow
```

Comment:
There are a few miscellaneous whitespace removals. My editor does that. I can raise a PR without them if necessary, but I think it's better to clear them out personally. 😄

Reply:
Definitely agree! 😁